test/functional/mempool_persist.py

```diff
@@ -72,20 +66,24 @@ def run_test(self):
         self.nodes = []
         self.nodes.append(start_node(0, self.options.tmpdir))
         self.nodes.append(start_node(1, self.options.tmpdir))
-        assert_equal(len(self.nodes[0].getrawmempool()), 5)
-        assert_equal(len(self.nodes[1].getrawmempool()), 0)
+        # Give bitcoind a second to reload the mempool
+        time.sleep(1)
+        assert wait_until(lambda: len(self.nodes[0].getrawmempool()) == 5)
+        assert wait_until(lambda: len(self.nodes[1].getrawmempool()) == 0)
```

Comment:
Does the `wait_until` accomplish anything here, or at line 80 below?  Either the mempool will be empty when this RPC is first invoked, in which case the wait_until will immediately return, or else it will have something -- in which case waiting and then failing later doesn't seem useful?